### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by KanoCode**
Add contact form to our website to enable users to interact with us

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,5 +175,76 @@ <h2>Multi-Post Stories</h2>
...
+    <section class="contact-form">
+      <div class="description">
+        <h1>Contact Us</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/mzbodazz">
```
Comment (andersonlebon) [inline index.html]: - [ ] Nice job on integrating the formspace service however you forgot to add the `method="POST"` attribute in your form that's why I am getting this 👇 👇  notification when submitting the form.
image

this 👇  is  how it show look like
```html
 <form action="https://formspree.io/f/mzbodazz" method="POST">
    ...content
 <form>

```

File: style.css [hunk 2]
```diff
@@ -303,3 +333,79 @@ h3 {
...
+  height: 50px;
+  margin: 12px;
+}
+
+textarea {
+  margin: 12px;
+  min-height: 148px;
+  width: 327px;
+}
+
```
Comment (andersonlebon) [inline style.css]: - [ ] One of the requirements says that you should stick to the design as much as you can. Kindly your inputs and the textarea miss padding around (`padding: 15px 16px`). check the difference here 👇  👇 

your design | the template
---|--
image | image

File: index.html [hunk 3]
```diff
@@ -171,5 +175,76 @@ <h2>Multi-Post Stories</h2>
...
+          required
+          name="message"
+          id="message"
+          placeholder="Enter text here"
+          maxlength="500"
+          
+
+        ></textarea>
+        <div id="contactButton">
+        <button type="submit">Submit</button>
```
Comment (andersonlebon) [inline index.html:217]: - [ ] On requirement states that `Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.` Kindly follow this requirement by adding the button interactions. here is the corresponding design 👇  
image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @KanoCode 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi @KanoCode 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**